backend/business-partner-agent/src/main/resources/log4j2.xml

```diff
@@ -4,9 +4,9 @@
         <Property name="PATTERN">%d{HH:mm:ss.SSS} [%t] %-5level %c{1} - %replace{%msg}{[\r\n]+}{}%n</Property>
     </Properties>
     <Appenders>
```

Comment:
The appenders section is there for a reason. Mainly to configure more than one appender and then select one through sysprops. But I guess this works as well.